infra.tf

```diff
@@ -1,18 +1,20 @@
 resource "aws_s3_bucket" "etcd_backups" {
   bucket_prefix = "${local.name}-etcd-backup"
-  acl           = "private"
   force_destroy = true
+}
 
-  server_side_encryption_configuration {
-    rule {
-      apply_server_side_encryption_by_default {
-        sse_algorithm = "aws:kms"
-      }
-    }
-  }
+resource "aws_s3_bucket_acl" "etcd_backups_acl" {
+  bucket = aws_s3_bucket.etcd_backups.id
+  acl    = "private"
+}
+
+resource "aws_s3_bucket_server_side_encryption_configuration" "etcd_backups_server_side_encryption_configuration" {
+  bucket = aws_s3_bucket.etcd_backups.id
 
-  versioning {
```

Comment:
Needs `aws_s3_bucket_versioning` instead of just disabling versioning.

Reply:
This [commit](https://github.com/drpebcak/terraform-rancher-workload-cluster/pull/3/commits/6617c609fbf5821755919ae6496111e64f36376d) should resolve this.